iceberg/iceberg-rest-server/src/main/java/org/apache/gravitino/iceberg/service/rest/IcebergTableOperations.java

```diff
@@ -195,21 +199,23 @@ public Response loadTable(
       @PathParam("table") String table,
       @DefaultValue("all") @QueryParam("snapshots") String snapshots,
       @HeaderParam(X_ICEBERG_ACCESS_DELEGATION) String accessDelegation) {
+    String catalogName = IcebergRestUtils.getCatalogName(prefix);
+    Namespace icebergNS = RESTUtil.decodeNamespace(namespace);
     boolean isCredentialVending = isCredentialVending(accessDelegation);
     LOG.info(
-        "Load iceberg table, namespace: {}, table: {}, access delegation: {}, "
+        "Load iceberg table, catalog: {}, namespace: {}, table: {}, access delegation: {}, "
```

Comment:
Can you please change all the "iceberg" to "Iceberg" here and below?

Reply:
updated